docs/output.md

```diff
@@ -108,44 +138,93 @@ The [bigWig](https://genome.ucsc.edu/goldenpath/help/bigWig.html) format is in a
+  * `counts_gene.txt` - gene expression estimates calculated by featureCounts.
+  * `counts_gene.txt.summary` - featureCounts gene level log file.
+  * `counts_transcript.txt` - transcript expression estimates calculated by featureCounts.
+  * `counts_transcript.txt.summary` - featureCounts transcript level log file.
+
+</details>
 
 *Documentation*:  
-[MultiQC](https://multiqc.info/docs/)
+[bambu](https://github.com/GoekeLab/bambu), [StringTie2](https://ccb.jhu.edu/software/stringtie/), [featureCounts](http://bioinf.wehi.edu.au/featureCounts/)
 
 *Description*:  
-*MultiQC* is a visualisation tool that generates a single HTML report summarising all samples in your project. Most of the pipeline QC results are visualised in the report and further statistics are available within the report data directory.  
+After genomic alignment, novel transcripts can be reconstructed using tools such as bambu and StringTie2. Quantification can then be performed on a more complete annotation based on the transcripts detected within a given set of samples. bambu performs both the reconstruction and quantification steps. An an alternative approach, we also provides an option to run StringTie2 to identify novel transcripts. However, when multiple samples are provided, quantification for multiple samples are not implemented explicitly in the software. Hence a second step is required to merge novel transcripts across multiple samples followed by quantification for both gene and transcripts using featureCounts. You can skip transcript reconstruction and quantification by providing the `--skip_quantification` parameter.
 
-Results generated by *MultiQC* for this pipeline collate QC from *FastQC*, *samtools flagstat*, *samtools idxstats* and *samtools stats*.
+## Differential expression analysis
 
-The pipeline has special steps which also allow the software versions to be reported in the *MultiQC* output for future traceability. For more information about how to use *MultiQC* reports, see <http://multiqc.info>.
+<details markdown="1">
+<summary>Output files</summary>
+
+* `<QUANTIFICATION_METHOD>/deseq2/deseq2.results.txt` - a `.txt` file that can contains differential expression results for genes.
+* `<QUANTIFICATION_METHOD>/dexseq/dexseq.results.txt` - a `.txt` file that can contains differential expression results for transcripts.
 
-*Output directories*:  
+</details>
+
+*Documentation*:
+[DESeq2](https://bioconductor.org/packages/release/bioc/html/DESeq2.html), [DEXSeq](https://bioconductor.org/packages/release/bioc/html/DEXSeq.html)
+
+*Description*:  
+If multiple conditions and multiple replicates are available then the pipeline is able to run differential analysis on gene and transcripts with DESeq2 and DEXSeq, respectively. These steps won't be run if you provide the `--skip_quantification` or `--skip_differential_analysis` parameters or if all of the samples in the samplesheet don't have the same fasta and GTF reference files.
```

Comment:
```suggestion
If multiple conditions and multiple replicates are available then the pipeline is able to run differential analysis on gene and transcripts with *DESeq2* and *DEXSeq*, respectively. These steps won't be run if you provide the `--skip_quantification` or `--skip_differential_analysis` parameters or if all of the samples in the samplesheet don't have the same fasta and GTF reference files.
```

Reply:
I don't think it matters as much if we leave these without emphasis. Note: we will also have to change this for all of the other tools in that document and possibly others too. Maybe we should leave things as they are?